packages/client-sdk-nodejs/src/config/middleware/experimental-metrics-logging-middleware.ts

```diff
@@ -105,6 +105,9 @@ export class ExperimentalMetricsLoggingMiddleware extends ExperimentalMetricsMid
   private static elu: EventLoopUtilization;
   private static isLoggingStarted = false;
   static numActiveRequests = 0;
+  // this is typed as any because JS returns a number for intervalId but
+  // TS returns a NodeJS.Timeout.
+  private static intervalId: any | null = null; // Store the interval ID
```

Comment:
I think you can turn off the eslint warning if you know you need to use `any` for this to work, something like so?

```
private static intervalId: any | null = null; // eslint-disable no-explicit-any
```